svgpathtools/paths2svg.py

```diff
@@ -240,15 +219,7 @@ def disvg(paths=None, colors=None,
     assert paths or nodes
     stuff2bound = []
     if viewbox:
-        if not isinstance(viewbox, str):
-            viewbox = '%s %s %s %s' % viewbox
-        if dimensions is None:
```

Comment:
yeah... looks like you didn't update fetch new changes before making this pull-request.